packages/help-center/src/hooks/use-open-article-in-help-center.ts

```diff
@@ -0,0 +1,23 @@
+import { HelpCenter } from '@automattic/data-stores';
+import { useDispatch as useDataStoreDispatch } from '@wordpress/data';
```

Comment:
I think we can just do `import { useDispatch } from '@wordpress/data';`, the "as" is used to differentiate in case we were both in need of a dispatch to "@wordpress/data" and to the Calypso store.